### ISSUE TITLE
Test out new layouts (internal/external)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#457]
Issue reported by **Julie Harold** at 2015-10-08 07:19:27 UTC
Time: 2015-10-08T07:16:22+00:00
Session ID: 
Problem Page URL: []()
Previous Page URL: [http://freereg2.freereg.org.uk/contacts/new](http://freereg2.freereg.org.uk/contacts/new)
Reported Issue:
Not a problem - just a couple of comments.  I really like the county summary in the database contents (is that new or have I just missed it all this  time?! ).

One suggestion - could be lose the ditto (") marks and actually have the place name - eg when a place has two churches the second entry has the place as ", similarly if there is a transcript as well as a register then they each get a "

Julie    

![Screenshot](http://www.freereg.org.uk)

**Comment by Sherlock21** [comment]
Yes to loose the (") marks - the style needs to be same as in the Active Place Names  view please.
and yes also to the summary style table - I had not realised by not picking a County, one got ALL - A good plan.

**Comment by PatReynolds** [comment]
Added to Intern task list: conduct formative user evaluation into potential changes from use of ditto marks to repeating text in the county summaries in the listing of county records

**Comment by Gingerygamer** [comment]
Survey of users conducted and results analyzed. Users prefer place names to ditto marks.

**Comment by PatReynolds** [comment]
Thanks - you should have a google hangout invite for now.

Cheers,

Pat

*- -*

*Dr Pat Reynolds*
Executive Director
Free UK Genealogy  <http://www.freeukgenealogy.org.uk/>
A charity registered in England and Wales, number 1096940, transitioning to
a Charitable Incorporated Organisation registered in England and Wales,
number 1167484
VAT registration: 233 0105 70

​+44 ​1904 541411​ +44 7943 145387

36 Albemarle Road, York, YO23 1ER, UK

**Comment by Captainkirkdawson** [comment]
Ditto marks removed

**Comment by Sherlock21** [comment]
My comment above about "Loose the ditto marks" did NOT amount to a vote to replace them with the full Place name and church name everywhere! That was not in the options on thins issue number.
And for that matter: where are the other votes? not on here are they?

**Comment by Captainkirkdawson** [comment]
See previous comment by person who conducted the survey of users.

**Comment by PatReynolds** [comment]
I will chase Meghan now for the final version of the reserch paper.

_\- -_

_Dr Pat Reynolds_
Executive Director
Free UK Genealogy  http://www.freeukgenealogy.org.uk/
A charity registered in England and Wales, number 1096940, transitioning to
a Charitable Incorporated Organisation registered in England and Wales,
number 1167484
VAT registration: 233 0105 70

​+44 ​1904 541411​ +44 7943 145387

36 Albemarle Road, York, YO23 1ER, UK

**Comment by PatReynolds** [comment]
Revisiting after some months.... Checking: Meghan tested ditto marks vs names, and was told that these were not the two comparitors needed.  Please confirm what needs comparison.

**Comment by PatReynolds** [comment]
Also, do we need to check for variation across listings and make changes/standardize across all for coords/transcribers, or is this to be decided on a case-by-case basis?

**Comment by Captainkirkdawson** [comment]
Pat
The ditto marks were removed sometime ago.

However the fundamental question on best layout remains the same. When we have a table where say the second and third column entries are the same. Do we display the content (A below)? Do we leave the content blank (B below)? or do we put as we did the ditto marks? or do we do something else
A
![display2](https://cloud.githubusercontent.com/assets/2874694/21246209/8478aef6-c2e4-11e6-8c0d-47d161a8d0f6.JPG)
B
![display1](https://cloud.githubusercontent.com/assets/2874694/21246220/945ba4cc-c2e4-11e6-9b9b-eb445940214f.JPG)
